### PR TITLE
Extraneous basemap labels layer

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes https://github.com/CartoDB/support/issues/963.

I couldn't reproduce the issue, even migrating maps from the editor, neither locally or in staging. We are almost sure this is a problem related to the migration between editor and builder, so I started to work on a different angle. Once we face this issue, the map itself will be able to autofix it, deleting those extra tiled layers.

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/data/layer-definitions-collection.js [hunk 1]
```diff
@@ -265,6 +281,7 @@ module.exports = Backbone.Collection.extend({
       silent: true,
       wait: true, // do not add/remove the layer unless it's created/removed/updated successfully
       success: function () {
+        this._sanitizeLabels();
```
Comment (ivanmalagon) [inline lib/assets/core/javascripts/cartodb3/data/layer-definitions-collection.js]: Why in labelsLayerOptions? This is only triggered if the basemap has labels, right? This won't get triggered if the user selects a non-labels basemap (correct if I'm wrong).

Comment (nobuti) [reply lib/assets/core/javascripts/cartodb3/data/layer-definitions-collection.js]: Fixed.

**Comment by ivanmalagon** [comment]
Tested with the attached map in the original ticket.
Great bugfix!!

By the way I've found a bug doing the acceptance related to layer but not this PR.

**Comment by ivanmalagon** [comment]
Shipped! :rocket:

**Pull request merged**